charts/fleet/Chart.yaml

```diff
@@ -8,7 +8,7 @@ version: v6.2.0
 home: https://github.com/fleetdm/fleet
```

Comment:
https://github.com/fleetdm/fleet/issues/23680 There is a callout that we should also be incrementing the chart version on line 7 (so for this PR v6.2.1) I will be adding a step to release to cover this going forward but it would be good to increment it manually in this PR